### PR TITLE
send iframe styling information to the embedded ccp

### PR DESCRIPTION
**Pull request by jeonglee**
**Motivation**:
tracking custom ccp usage

**Testing**:
local embedded ccp

*Description of changes:*
use postMessage to send iframe styling information to ccp

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/core.js [hunk 1]
```diff
@@ -426,6 +426,19 @@
     iframe.style = "width: 100%; height: 100%";
     containerDiv.appendChild(iframe);
 
+    // Let CCP know if iframe is visible
+    iframe.onload = setTimeout(function() {
```
Comment (limamazon) [inline src/core.js:430]: Can we follow the structure of other iframe messages? i.e.
```
{event: eventName, data: data}
```

Comment (jeonglee) [reply src/core.js]: i decided to use the iframe conduit since that would be cleaner

File: src/core.js [hunk 2]
```diff
@@ -436,6 +436,18 @@
...
+      var style = window.getComputedStyle(iframe, null);
+      var data = {
+        display: style.display,
+        offsetWidth: iframe.offsetWidth,
+        offsetHeight: iframe.offsetHeight,
+        clientRectsLength: iframe.getClientRects().length
+      };
+      conduit.sendUpstream(connect.EventType.IFRAME_STYLE, data);
```
Comment (jolim24601) [inline src/core.js:448]: Does this `EventType` exist?

Comment (jeonglee) [reply src/core.js:448]: hmm I added it in event.js but I guess it didn't get committed. I tested and made sure it worked / i'll add it back in

**Pull request merged**